testsupport/signup_request.go

```diff
@@ -112,6 +112,39 @@ func (r *SignupRequest) Resources() (*toolchainv1alpha1.UserSignup, *toolchainv1
 	return r.userSignup, r.mur
 }
 
+func (r *SignupRequest) GetSignupResponse(t *testing.T) map[string]interface{} {
+	hostAwait := r.awaitilities.Host()
+	userIdentity := &commonauth.Identity{
+		ID:       r.identityID,
+		Username: r.username,
+	}
+	claims := []commonauth.ExtraClaim{commonauth.WithEmailClaim(r.email)}
+	if r.originalSub != "" {
+		claims = append(claims, commonauth.WithOriginalSubClaim(r.originalSub))
+	}
+	if r.userID != "" {
+		claims = append(claims, commonauth.WithUserIDClaim(r.userID))
+	}
+	if r.accountID != "" {
+		claims = append(claims, commonauth.WithAccountIDClaim(r.accountID))
+	}
+
+	var err error
+	r.token, err = authsupport.NewTokenFromIdentity(userIdentity, claims...)
+	require.NoError(t, err)
+
+	queryParams := map[string]string{}
+	if r.noSpace {
+		queryParams["no-space"] = "true"
+	}
+
+	// Call the signup GET endpoint
+	response := invokeEndpoint(t, "GET", hostAwait.RegistrationServiceURL+"/api/v1/signup",
+		r.token, "", http.StatusOK, queryParams)
+
+	return response
+}
```

Comment:
why do we need to execute this here? How is it different compared to what we have in `Execute()` method?
If we are missing test that verifies the response of the `GET` request, then it should go into the `registration_service_test.go` not here. However, I believe that we have all needed verification already there, right?